src/main.cc

```diff
@@ -0,0 +1,88 @@
+#ifdef ONPC
+#include "../algo/debug.h"
+#else
+#define debug(...) 42
+#endif
+#include <cassert>
+#include <cstdint>
+#include <utility>
+#include <vector>
+
+template<typename T>
+using Pair = std::pair<T, T>;
```

Comment:
The idea was not to just alias `std::pair`, but to make pair's parts have more clear names  instead of first second.